packages/wrangler/src/dev/dev-vars.ts

```diff
@@ -0,0 +1,38 @@
+import * as fs from "node:fs";
+import * as path from "node:path";
+import dotenv from "dotenv";
+import { logger } from "../logger";
+import type { Config } from "../config";
+
+/**
+ * Get the Worker `vars` bindings for a `wrangler dev` instance of a Worker.
+ *
+ * The `vars` bindings can be specified in the `wrangler.toml` configuration file.
+ * But "secret" `vars` are usually only provided at the server -
+ * either by creating them in the Dashboard UI, or using the `wrangler secret` command.
+ *
+ * It is useful during development, to provide these types of variable locally.
+ * When running `wrangler dev` we will look for a file called `.dev.vars`, situated
+ * next to the `wrangler.toml` file (or in the current working directory if there is no
+ * `wrangler.toml`).
+ *
+ * Any values in this file, formatted like a `dotenv` file, will add to or override `vars`
+ * bindings provided in the `wrangler.toml`.
+ */
+export function getVarsForDev(config: Config): Config["vars"] {
+  const configDir = path.resolve(path.dirname(config.configPath ?? "."));
+  const devVarsPath = path.resolve(configDir, ".dev.vars");
```

Comment:
How do you feel about also having a command line arg for `dev`, called say `--dev-vars` that lets you specify a path to a `.dev.vars` file? It strikes me that it's possible that this file may have secrets, and one way to prevent secrets from accidentally leaking is to have it outside the project repo itself. Further, you may want to try with different sets of secrets/vars (which isn't particularly recommended, but folks do it anyway).

Reply:
Yeah. I spent some time thinking about this and couldn't decide what the best course of action would be.

So I opted for not adding it, since it would be easy to add in 2.1 but hard to remove.

Let's consider if for post-2.0.0?